cnxauthoring/tests/test_models.py

```diff
@@ -22,6 +22,124 @@ class ModelUtilitiesTestCase(unittest.TestCase):
+    def test_revise_content(self):
+        archive_url = "http://example.com"
+        settings = {'archive.url': archive_url}
+
+        from ..models import DEFAULT_LICENSE
+        content_id = 'uuid'
+        content_title = 'title'
+        archived_data = {
+            'id': content_id,
+            'version': '1',
+            'license': DEFAULT_LICENSE.__json__(),
+            'title': content_title,
+            'authors': [{}, {}, {}],
+            'maintainers': [{}, {}, {}],
+            'publishers': [{}, {}, {}],
+            'licensors': [{}, {}, {}],
+            'translators': [{}, {}, {}],
+            'editors': [{}, {}, {}],
+            'illustrators': [{}, {}, {}],
+            'created': 'awhile ago',
+            'revised': 'just now',
+            }
+
+        url = "{}/contents/{}.json".format(archive_url, content_id)
+        faux_response_body = json.dumps(archived_data)
+        httpretty.register_uri(httpretty.GET, url,
+                               body=faux_response_body, status=200)
+
+        request = pyramid_testing.DummyRequest()
+        request.registry = mock.Mock()
+        request.registry.settings = settings
+
+        from ..models import revise_content
+        document_as_dict = revise_content(request, id=content_id)
+
+        expected = archived_data.copy()
+        role_attrs = ('authors', 'licensors', 'editors', 'illustrators',
+                      'maintainers', 'publishers', 'translators',)
+        for role_attr in  role_attrs:
+            expected[role_attr] = [{'has_accepted':True},{'has_accepted':True},{'has_accepted':True}]
+        expected['revised'] = None
+        self.assertEqual(document_as_dict, expected)
+
+    @httpretty.activate
+    def test_revise_content_upgrades_license(self):
+        archive_url = "http://example.com"
+        settings = {'archive.url': archive_url}
+
+        from ..models import LICENSES, DEFAULT_LICENSE
+        license = [l for l in LICENSES if l.url.find('by/3') >= 0][0]
+
+        content_id = 'uuid'
+        content_title = 'title'
+        archived_data = {
+            'id': content_id,
+            'version': '1',
+            'license': license.__json__(),
+            'title': content_title,
+            'publishers': [{}, {}, {}],
+            }
+
+        url = "{}/contents/{}.json".format(archive_url, content_id)
+        faux_response_body = json.dumps(archived_data)
+        httpretty.register_uri(httpretty.GET, url,
+                               body=faux_response_body, status=200)
+
+        request = pyramid_testing.DummyRequest()
+        request.registry = mock.Mock()
+        request.registry.settings = settings
+
+        from ..models import revise_content
+        document_as_dict = revise_content(request, id=content_id)
+
+        self.assertEqual(document_as_dict['license']['version'],
+                         DEFAULT_LICENSE.version)
+        self.assertEqual(document_as_dict['license']['upgraded'],True)
+
+
```

Comment:
pep8 "too many blank lines (2)"

Only use two lines between function and class definitions.